Makefile

```diff
@@ -37,7 +37,12 @@ post-wsl:
 install-code-plugins:
 	code --install-extension aaron-bond.better-comments \ 
 	code --install-extension dbaeumer.vscode-eslint \
-	code --install-extension esbenp.prettier-vscode
+	code --install-extension esbenp.prettier-vscode \
+	code --install-extension eamodio.gitlens \
+	code --install-extension formulahendry.auto-rename-tag \
```

Comment:
Remove `gitlens`, `auto-rename-tag` and `gotestexplorer`. You may add a recommended plugins section to the README beneath the recommended editor section and include gitlens there. We want the `install-code-plugins` phony to be installing only essential plugins for development. GitLens is a proprietary one so I would prefer that we don't include it in the plugins installation.